tests/rest/client/v2_alpha/test_register.py

```diff
@@ -302,6 +308,292 @@ def test_request_token_existing_email_inhibit_error(self):
 
         self.assertIsNotNone(channel.json_body.get("sid"))
 
```

Comment:
I'm open to advice for how to reduce the repetition in these four tests, of which the only difference is changing config options and checking whether relevant methods were called once or twice.

Reply:
Create a helper that does the actual test and takes some input parameters + the number of times to assert the mock has been called? Then each test essentially becomes:

```python
@override_config(...)
def test_foo(self):
    self._helper_method(...)
```

Reply:
Hmm, yes, I would normally do this but shied away from it as there's a couple other checks required, but thinking about it those can be abstracted pretty easily actually.

Let me give it a shot.

Reply:
And that was actually quite simple!